microcosm_flask/tests/conventions/test_command.py

```diff
@@ -0,0 +1,130 @@
+"""
+Command operation tests.
+
+"""
+from json import dumps, loads
+from hamcrest import (
+    assert_that,
+    equal_to,
+    is_,
+)
+
+from marshmallow import fields, Schema
+from microcosm.api import create_object_graph
+
+from microcosm_flask.conventions.encoding import dump_response_data, load_request_data
+from microcosm_flask.conventions.registry import request, response
+from microcosm_flask.operations import Operation
+
+
+class CommandArgumentSchema(Schema):
+    value = fields.String(required=True)
+
+
+class CommandResultSchema(Schema):
+    result = fields.Boolean(required=True)
+    value = fields.String(required=True)
+
+
+def make_command(graph, request_schema, response_schema):
+    """
+    Create an example command route.
```

Comment:
On a more philosophical note, why make command endpoints as opposed to generating virtual RESTful resources?

 An example is https://globality.readme.io/docs/submit-scopeofwork, where the action is POST `.../submission` and it "creates" a "submission", which (as implementation details internal to the service) changes the state of the associated object and runs any needed side effects for the business.